src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -278,12 +279,19 @@ private static void syncConfig(boolean load)
 
         // Client-Side only properties
         propOrder = new ArrayList<String>();
+
         prop = config.get(Configuration.CATEGORY_CLIENT, "replaceVanillaBucketModel", Boolean.FALSE,
                 "Replace the vanilla bucket models with Forges own dynamic bucket model. Unifies bucket visuals if a mod uses the Forge bucket model.");
         prop.setLanguageKey("forge.configgui.replaceBuckets").setRequiresMcRestart(true);
         replaceVanillaBucketModel = prop.getBoolean(Boolean.FALSE);
         propOrder.add(prop.getName());
 
+        prop = config.get(Configuration.CATEGORY_CLIENT, "zoomInMissingModelTextInGui", Boolean.TRUE,
```

Comment:
Dang, turns out there's 3 places that store the default value for the config property, and not 2.